assets/js/hooks/session.js

```diff
@@ -1022,7 +1022,9 @@ const Session = {
       const focusedSectionId = this.getSectionIdByFocusableId(this.focusedId);
 
       if (focusedSectionId) {
-        this.setFocusedEl(focusedSectionId, { scroll: true });
+        setTimeout(() => {
+          this.setFocusedEl(focusedSectionId, { scroll: true });
+        }, 200);
```

Comment:
What does the timeout address?

Reply:
yesterday I tried to expand a large, collapsed notebook and it wasn't scrolling to the selected section. Today I tried it again and now it works without the timeout :-D
so I think I just forgot to revert some changes which leads to that behavior.

> the unexpected behaviour is that scrolling to a section doesn't highlight it in the sidebar right?
Exactly. The problem is if the sections are collapsed, they are very short. But as you said I think we can ignore this :)